BUILD

```diff
@@ -161,6 +161,13 @@ pkg_tar(
     visibility = ["//:__subpackages__"],
 )
 
+pkg_tar(
```

Comment:
Clever.  This target could use a comment to explain that \@maven is created as a side effect of rules_jvm_external. That way people won't go hunting for it in our own sources.